addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -287,49 +287,62 @@ body .modal {
                 min-height: 300px;
             }
 
-            .existing-attachments {
-                width: 100%;
+            .o_we_images {
+                .o_existing_attachment_cell {
+                    // give enough space for the hover buttons
+                    min-width: 64px;
+                    img {
+                        // prevent hover/select border from flickering
+                        border: 1px solid transparent;
+                        // prevent url images from being too big
+                        max-height: 192px;
+                    }
+                    &:hover img {
+                        border-color: $card-border-color;
+                        box-shadow: 0px 0px 2px 2px $card-border-color;
+                    }
+                    &.o_we_attachment_selected img {
+                        border-color: $o-brand-primary;
+                        box-shadow: 0px 0px 2px 2px $o-brand-primary;
+                    }
+                }
+            }
+
+            .o_we_documents {
+                .o_existing_attachment_cell {
+                    img {
+                        // prevent hover/select border from flickering
+                        border: 1px solid transparent;
+                    }
+                    &:hover img {
+                        border-color: $card-border-color;
+                        box-shadow: 0px 0px 2px 2px $card-border-color;
+                    }
+                    &.o_we_attachment_selected {
+                        border-color: $o-brand-primary;
+                        box-shadow: 0px 0px 2px 2px $o-brand-primary;
```

Comment:
The problem here is that the DOM is different, for the image case we want to put the border around the image `img`, but for the document we want the border around the whole area and not just around the small image.

The current diff is actually bad because I forgot for some rules to remove the `img` from my copy/paste into documents.

How to handle it then?

Reply:
From worst to best:
- Using a scss mixin
- Using an extend of scss placeholder
- Reviewing the DOM to add the targetable class on the img/the container depending on the case

Reply:
Done with best option.